pb/summary/summary.proto

```diff
@@ -122,6 +122,12 @@ message HealthinessInfo {
   repeated float previous_flakiness = 5;
 }
 
+// Information about alerts that have been sent
+message AlertingData {
+  // Seconds since epoch at which an email was last sent
+  double last_email_timestamp = 1;
```

Comment:
This should be a timestamp instead of a double, if possible

Reply:
How difficult will it be to migrate from a double to a timestamp?

The equivalent entry we're trying to migrate this from, [`last_alert_mail_time`](https://github.com/GoogleCloudPlatform/testgrid/blob/4e72272c6d8e329a458c686056fc7d19fe12c786/pb/state/state.proto#L167), is a double for legacy reasons.

Reply:
Should be relatively easy; the timestamp message just has 'seconds' and 'nanos' as fields, so instead of using the field directly, you'd use the timestamp.seconds instead (maybe multiplied by some factor because I don't remember if we store seconds, milliseconds, or something else ><)

Reply:
Done